tests/json/retryable_writes/bulkWrite.json

```diff
@@ -629,7 +629,132 @@
         "result": {
           "deletedCount": 0,
           "insertedIds": {
-            "0": 2
+            "1": 2
```

Comment:
This is from [SPEC-1064](https://jira.mongodb.org/browse/SPEC-1064), although it's not relevant to libmongoc since you don't expose `insertedIds`.